server/services/involvedStaffService.ts

```diff
@@ -81,7 +81,14 @@ export class InvolvedStaffService {
 
       if (report.status === ReportStatus.COMPLETE.value) {
         logger.info(`There are now pending statements on : ${reportId}, moving from 'COMPLETE' to 'SUBMITTED'`)
-        await this.incidentClient.changeStatus(reportId, ReportStatus.COMPLETE, ReportStatus.SUBMITTED, client)
+        // TODO provide real username
+        await this.incidentClient.changeStatus(
```

Comment:
Still a bit unsure about this. 

These are automatic transitions that can happen as a side effect  when a staff member is added/removed to the report.

We could look at passing the staff member who triggered the action - though its not like a conscious action the user did